cmd/crowdsec/main.go

```diff
@@ -308,7 +308,8 @@ func main() {
 
 	if exitCode, err := Serve(cConfig); err != nil {
 		if err != nil {
-			log.Errorf(err.Error())
+			log.Error(err)
+			fmt.Fprintln(os.Stderr, err)
```

Comment:
as discussed, it overlaps with the existing code that adds a hook on logger : `log.AddHook` (cf. main)